src/_scss/layouts/default/header/nav/mobile/_nav.scss

```diff
@@ -1,48 +1,61 @@
-.nav-content {
-    margin-top: rem(50);
-    ul {
+.mobile-nav-content {
+    ul.mobile-nav-content__list {
         @include unstyled-list;
+        margin-top: rem(50);
 
-        li {
+        li.mobile-nav-content__list-item {
             text-align: center;
-
-            a.nav-link {
-                color: $color-gray;
-                font-size: rem(30);
-                line-height: rem(36);
-                opacity: 0.6;
-
-                &:active, &:hover {
-                    text-decoration: none;
-                    opacity: 1;
+            &.mobile-nav-content__list-item_no-phone {
+                @media screen and (max-width: 450px) {
+                    display: none;
                 }
             }
+        }
 
-            button.nav-button {
-                @include button-unstyled;
-                color: $color-gray;
-                font-size: rem(30);
-                line-height: rem(36);
-                opacity: 0.6;
-            }
+        a.mobile-nav-content__link {
+            color: $color-gray;
+            font-size: rem(30);
+            line-height: rem(36);
+            opacity: 0.6;
 
-            .nav-link-decorator {
-                border-bottom: 2px solid $color-gray;
-                width: rem(50);
-                margin-left: auto;
-                margin-right: auto;
-                margin-top: rem(20);
-                margin-bottom: rem(20);
-                opacity: 0.6;
+            &:active, &:hover {
+                text-decoration: none;
+                opacity: 1;
             }
+        }
 
-            @import "./_dropdown";
+        button.mobile-nav-content__glossary {
+            @include button-unstyled;
 
-            &.no-phone {
-                @media screen and (max-width: 450px) {
-                    display: none;
-                }
+            color: $color-gray;
+            font-size: rem(30);
+            line-height: rem(36);
+            opacity: 0.6;
+
+            &:active, &:hover {
+                opacity: 1;
             }
         }
+
+        button.nav-button {
```

Comment:
Two questions – 1) Should this be something like `button.mobile-nav-content__nav-button`? and 2) Is this style even being used anymore?

Reply:
Not being used any more, deleted